Update README (#201)
SevenTV/chatterino7: README.md

![alt text](https://fourtf.com/img/chatterino-icon-64.png)

> Chatterino 2 [![GitHub Actions Build (Windows, Ubuntu, MacOS)](https://github.com/SevenTV/chatterino7/workflows/Build/badge.svg?branch=master)](https://github.com/SevenTV/chatterino7/actions?query=workflow%3ABuild+branch%3Amaster) [![Chocolatey Package](https://img.shields.io/chocolatey/v/chatterino7?include_prereleases)](https://chocolatey.org/packages/chatterino7)

Reviewer: ```suggestion
Chatterino 7 [![GitHub Actions Build (Windows, Ubuntu, MacOS)](https://github.com/SevenTV/chatterino7/workflows/Build/badge.svg?branch=chatterino7)](https://github.com/SevenTV/chatterino7/actions?query=workflow%3ABuild+branch%3Achatterino7) [![Chocolatey Package](https://img.shields.io/chocolatey/v/chatterino7?include_prereleases)](https://chocolatey.org/packages/chatterino7)
```

============
## Chatterino7

> Chatterino7 is a fork of Chatterino 2 that says in line with Chatterino 2 but adds features from 7TV, that aren't supported by regular Chatterino.

Reviewer: ```suggestion
Chatterino7 is a fork of Chatterino 2. This fork mainly contains features that aren't accepted into Chatterino 2, most notably 7TV subscriber features.
```

> ### Features of Chatterino7

Reviewer: Maybe this could be a table like this:

|Feature| Chatterino 7 | Chatterino 2 |
|---|:---:|:---:|
| 7TV Global Emotes | ✅ |✅ |
| 7TV Channel Emotes | ✅ |✅ |
| 7TV Emote Updates | ✅ |✅ |
| 7TV Badges | ✅ | ✅ * |
| 7TV Personal Emotes | ✅ | ❌  |
| 7TV Name Paints | ✅ | ❌  |
| 7TV Avatars | ✅ | ❌  |
| 4x Emotes | ✅ | ❌  |

*: Chatterino 2 does not yet support the new cosmetics system.

Author: 4x Images*

> - 7TV Paints

Reviewer: ```suggestion
- 7TV Name Paints
```

> - 7TV Personal Emotes

Reviewer: ```suggestion
- 7TV Personal Emotes (_currently_: nighly only)
```

Reviewer 2: :nerd_face: you spelled nightly wrong

- 7TV Animated Profile Avatars

> - 4x Link Extensions(7TV and FFZ)

Reviewer: ```suggestion
- 4x Images (7TV and FFZ)
```

### Screenshots

> ![Example of 4x](https://kappa.lol/UpjGe)

Reviewer: I'm not sure if this section should be included here at all. If it stays, the image should be hosted on GitHub.

### Downloads

Stable Builds can be downloaded [here](https://github.com/SevenTV/chatterino7/releases)
Download Nightly Chatterino7 build over [here](https://github.com/SevenTV/chatterino7/releases/tag/nightly-build)

> Windows users can install Dankerino [from Chocolatey](https://chocolatey.org/packages/chatterino7).

Reviewer: ```suggestion
**Stable builds** can be downloaded from the [releases section](https://github.com/SevenTV/chatterino7/releases).

To test new features, you can download the **nighly build** [here](https://github.com/SevenTV/chatterino7/releases/tag/nightly-build).

Windows users can install Chatterino 7 [from Chocolatey](https://chocolatey.org/packages/chatterino7).
```

### Discord 

> If you have any issues or want to join the community you can join the link here: <https://discord.com/invite/7tv>

Reviewer: ```suggestion
### Issues

If you have issues such as crashes or weird behaviour regarding 7TV features, report them [in the issue-section](https://github.com/SevenTV/chatterino7/issues). If you have issues with other features, please report them [in the upstream issue-section](https://github.com/Chatterino/chatterino2/issues).

### Discord 

If you don't have a GitHub account and want to report issues or want to join the community you can join the official 7TV Discord using the link here: <https://discord.com/invite/7tv>.
```

> ## Original Chatterino 2 readme

Reviewer: ```suggestion
## Original Chatterino 2 Readme
```
